rfcs/51-identify-go-storage-error.md

```diff
@@ -0,0 +1,71 @@
+---
+author: xxchan <xxchan22f@gmail.com>
+status: draft
+updated_at: 2021-05-10
+---
+
+# AOS-51: Identify go-storage Error
```

Comment:
`Identify go-storage Error` is a not good proposal name to me. How about `Distinguish Errors by IsAosError`?